test/e2e/mock-e2e.js

```diff
@@ -671,17 +670,14 @@ async function setupMocking(
 
   await server
     .forGet('https://min-api.cryptocompare.com/data/pricemulti')
-    .withQuery({ fsyms: 'ETH,MON', tsyms: 'usd' })
+    .withQuery({ fsyms: 'ETH', tsyms: 'usd' })
     .thenCallback(() => {
       return {
         statusCode: 200,
         json: {
           ETH: {
             USD: ethConversionInUsd,
           },
-          MON: {
-            USD: monConversionInUsd,
-          },
```

Comment:
Removing this as we've optimised the `useCurrencyRatePolling` to not poll all networks. Only popular networks (which do not include testnets) and selected network.